tests/aws/services/apigateway/test_apigateway_http.py

```diff
@@ -42,23 +43,36 @@ def add_http_integration_transformers(snapshot):
 @markers.aws.validated
 @markers.snapshot.skip_snapshot_verify(
     paths=[
-        "$..content.headers.accept-encoding",  # TODO: We add an extra space when adding this header
+        # TODO: shared between HTTP & HTTP_PROXY
+        "$..content.headers.x-amzn-trace-id",
+        "$..content.headers.x-forwarded-for",
+        "$..content.origin",
+        "$..headers.server",
+        # TODO: for HTTP integration only: requests (urllib3) automatically adds `Accept-Encoding` when sending the
+        #  request, seems like we cannot remove it
+        "$..headers.accept-encoding",
+        "$..headers.x-amz-apigw-id",
+        # TODO: only missing for HTTP_PROXY
+        "$..headers.x-amzn-remapped-x-amzn-requestid",
+        "$..headers.x-amzn-trace-id",
+        # TODO: only missing for HTTP
+        "$..headers.x-amzn-requestid",
+        "$..headers.x-amzn-trace-id",
```

Comment:
nit: seems like `"$..headers.x-amzn-trace-id"` is missing for both.

Reply:
good catch, thanks!